packages/cli/src/auth/auth.service.ts

```diff
@@ -54,7 +54,7 @@ export class AuthService {
 				try {
 					req.user = await this.resolveJwt(token, res);
 				} catch (error) {
-					if (error instanceof JsonWebTokenError || error instanceof AuthError) {
+					if (error instanceof TokenExpiredError || error instanceof AuthError) {
```

Comment:
In this case it's better to check against `JsonWebTokenError` so we clear the cookie if the token is invalid (e.g. signing secret has changed). The case in the `resolvePasswordResetToken` specifially logs `Reset password token expired` so catching the more specific `TokenExpiredError` produces correct log output.